minecraft_servers/pluginstube/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "server_name": "pluginstube",
+  "nice_name": "PluginStube",
+  "direct_ip": "pluginstube.net",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://pluginstube.net",
+    "web_shop": "https://shop.pluginstube.net",
+    "web_support": "https://pluginstube.net/support",

```

Comment:
Page not found.

Reply:
Hey i have fixed the https://pluginstube.net/support domain

and compressed the background image ;)